benches/end2end.rs

```diff
@@ -46,7 +46,8 @@ fn end2end_benchmark(c: &mut Criterion) {
         .sample_size(10);
 
     let limit = 1_000_000_000;
-    let lang_vesta = Lang::<pasta_curves::Fq, Coproc<pasta_curves::Fq>>::new();
+    let lang_vesta =
+        Lang::<pasta_curves::pallas::Scalar, Coproc<pasta_curves::pallas::Scalar>>::new();
```

Comment:
Sorry, but this really needs to be called `lang_pallas`. The idea behind changing the field name used was to make that more obvious. Maybe the best thing is to fix the equivalent in `end2end` as well. It's just too confusing to have such explicit labels be inconsistent in the codebase, since that will lead to people wrongly anchor understanding/knowledge in ways we will eventually regret.